popup.js

```diff
@@ -13,6 +13,42 @@ const cardWork = [
+    longdescription: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    technologies: ['html', 'css', 'javaScript'],
+    linkToLive: 'https://github.com/emolio',
+    linkToSource: 'https://emarkees.github.',
+    cardNo: '#one',
+  },
+  {
+    name: 'Tonic',
+    Cartegories: ['CANOPY', 'Back End Dev', '2015'],
+    featuredImage: './Scr/images/professional.png',
+    shortDescription: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    description: ' Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    longdescription: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    technologies: ['html', 'css', 'javaScript'],
+    linkToLive: 'https://github.com/emolio',
+    linkToSource: 'https://emarkees.github.',
+    cardNo: '#one',
+  },
+  {
+    name: 'Multiple Stories',
+    Cartegories: ['CANOPY', 'Back End Dev', '2015'],
+    featuredImage: './Scr/images/nature.png',
+    shortDescription: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    description: ' Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    longdescription: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    technologies: ['html', 'css', 'javaScript'],
+    linkToLive: 'https://github.com/emolio',
+    linkToSource: 'https://emarkees.github.',
+    cardNo: '#one',
+  },
 ];
 
```

Comment:
- [ ]  Kindly check the last reviewer's comment and make changes, currently, the popup menu doesn't have a button for closing it.
- [ ] Another important thing to note is, the cards have a fixed width which makes them look smaller on the desktop version.

![popmenu](https://user-images.githubusercontent.com/53124548/211178663-81fd2b84-3104-44ea-ae1e-b6e473549513.png)

![close](https://user-images.githubusercontent.com/53124548/211178672-7c0e26f5-81b1-43ef-bd61-2fabe71534de.png)